src/main.js

```diff
@@ -1,89 +1,89 @@
 import http from 'http';
 import socketio from 'socket.io';
-import Monopoly from "./monopoly";
+import Monopoly from './monopoly';
 
 const server = http.createServer();
 const io = socketio(server);
 const monopolyGame = new Monopoly();
 
 io.on('connect', (socket) => {
```

Comment:
OK, looked at it and understand it.  this is my next thing im working on when it isn't 1AM